playwright/.auth/admin.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
I think that this file should probably be in `.gitignore`? It contains some sensitive auth stuff that we probably don't want to commit prublicly.